src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/service/MigrationQueueService.kt

```diff
@@ -52,23 +55,6 @@ class MigrationQueueService(
     }
   }
 
-  suspend fun <T : Enum<T>> sendMessage(message: T, context: SynchronisationContext<*>) {
```

Comment:
this did feel weird - a migration service using a synchronisation context, but turns out it wasn't used